src/icons/states/SecurityBlockedState.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray10, bdlGray50, white } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    strokeColor?: string,
```

Comment:
I think we're trying to avoid styling via props for icons, but I'm less sure about states.